BoxContentSDK/BoxContentSDK/Operations/BOXStreamOperation.m

```diff
@@ -144,12 +144,12 @@ - (void)abortWithError:(NSError *)error
 
 #pragma mark - BOXURLSessionTaskDelegate
 
-- (void)sessionTask:(NSURLSessionTask *)sessionTask didFinishWithResponse:(NSURLResponse *)response error:(NSError *)error
+- (void)processIntermediateResponse:(NSURLResponse *)response
 {
-    [super sessionTask:sessionTask didFinishWithResponse:response error:error];
-
-    if (error != nil &&
-        error.code == BOXContentSDKAPIErrorAccepted) {
+    [super processIntermediateResponse:response];
+    
+    if (self.error != nil &&
+        self.error.code == BOXContentSDKAPIErrorAccepted) {
```

Comment:
if we decide to remove processIntermediateResponse per https://github.com/box/box-ios-sdk/pull/333#pullrequestreview-24827686, we might want to keep it as before